### DISCUSSION TITLE
Help setting up STIX Two

### DISCUSSION THREAD

**Discussion by ararunaufc** [typst/typst#569] [Q&A]
Hey, guys.

I'm trying to use the STIX Two font set for an academic document, but I can't seem to make it fully work.

I'm using the CLI version `2c735294` (build from the GitHub repo), and this is the setting up:

```typst
set text(
    font: ("STIX Two Text", "Times New Roman", "serif"),
    size: 12pt,
    lang: "pt",
    region: "BR",
    kerning: true
)
```

The problem I get is that for `strong` and `emph` Typst is using the `Linux Libertine` fallback. I've already checked that the CLI can find the "STIX Two Text" font and that all the different styles are defined to have the same name (bold, italic, bold-italic etc. are in different `.ttf` files).

I also tried using `show strong: set text(font: ("STIX Two Text Bold"))`, but it does not work exactly because there is no such font name defined.

Is this a bug or am I doing it wrong?

Thanks.

**Comment by ararunaufc** [comment]
Just so you know, the fonts I'm talking about are the ones from [here](https://github.com/stipub/stixfonts), from the latest release.

**Reply by mareikep** [reply]
Had the same problem, typst fonts --variants only showed Italic and Normal for STIX Two Text, even though more variants are installed on the system and can be used properly by other applications. This issue seems to be discussed here https://github.com/typst/typst/issues/751 and here https://github.com/typst/typst/issues/576, though.  Guess we just hope the bug is fixed in the next release and use a different font until then.
